src/test/ui/traits/issue-75627.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0412]: cannot find type `T` in this scope
+  --> $DIR/issue-75627.rs:3:26
+   |
+LL | unsafe impl Send for Foo<T> {}
+   |                          ^ not found in this scope
```

Comment:
Should we suggest using `unsafe impl<T> Send for Foo<T>`?

Reply:
Ah you're right, I should leave the help message

Reply:
This doesn't actually appear to be introduced because of this change, so I'm going to leave it as is.

Reply:
@estebank Any idea what causes this regression?